<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_39">
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="showLabel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="creature_comboBox">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="creature_label">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Creature</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="show_label">
       <property name="text">
        <string>showLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="puppet_label">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Puppet</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Default Types</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="default_Component_listWidget">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addDefaultComponent_pushButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Component Types</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListWidget" name="newComponent_listWidget">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>95</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>95</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>AimComponent</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FaceComponent</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FKComponent</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LimbComponent</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SpineComponent</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SplineComponent</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addNewComponent_pushButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add New</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Builder Components</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QListWidget" name="component_listWidget">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="editComponent_pushButton">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Edit Component</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeComponent_pushButton">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Remove Component</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buildComponent_pushButton">
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Build</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buildAllComponents_pushButton">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Build All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteComponent_pushButton">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Anim Files</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="newFile_lineEdit"/>
           </item>
           <item>
            <widget class="QListWidget" name="newFile_listWidget">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="addFile_pushButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>10000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Add File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeFile_pushButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100000</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Remove File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportSelectedFiles_pushButton">
             <property name="text">
              <string>Export Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportAllFiles_pushButton">
             <property name="text">
              <string>Export All Files</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="deleteItems_lineEdit">
           <property name="text">
            <string>Delete Lists</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QListWidget" name="deleteItems_listWidget">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ContiguousSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QPushButton" name="addDeleteItems_pushButton">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Add Items</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeDeleteItems_pushButton">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Remove Items</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="loadPuppetFromScene_pushButton">
       <property name="text">
        <string>Load Puppet from Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save Builder to Puppet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBuilder_pushButton">
       <property name="text">
        <string>Export Builder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
